src/js/utils/pagination.js

```diff
@@ -0,0 +1,53 @@
+import { useCallback, useEffect, useState } from 'react';
+
+export const normalizeShow = (showProp, context) => {
+  let page;
+  let showItem;
+
+  if (['dataTable', 'list'].includes(context)) {
```

Comment:
Since we only call this from DataTable and List, can we just remove `context`?

Reply:
Yes, will update. Context was lingering from a previous iteration.

Reply:
Done.